bool read_robot_state
---
float64[] joint_positions
bool success
